css/style.css

```diff
@@ -0,0 +1,31 @@
+*{margin: 0;padding: 0;box-sizing: border-box;}
```

Comment:
Please format your code. Break which css property to another line. It is very hard to read and understand otherwise.